commands/user.go

```diff
@@ -969,19 +970,24 @@ func promoteGuestToUserCmdF(c client.Client, _ *cobra.Command, userArgs []string
 }
 
 func demoteUserToGuestCmdF(c client.Client, _ *cobra.Command, userArgs []string) error {
+	var errs error
 	for i, user := range getUsersFromUserArgs(c, userArgs) {
 		if user == nil {
-			printer.PrintError(fmt.Sprintf("can't find user '%v'", userArgs[i]))
+			errStr := fmt.Sprintf("can't find user '%v'", userArgs[i])
```

Comment:
We use `%s` for strings. Also it would make sense to use `fmt.Errorf` here and print `err.Error()`.

Reply:
updated